services/slam/fake/slam_test.go

```diff
@@ -34,6 +39,31 @@ func TestFakeSLAMPosition(t *testing.T) {
 	test.That(t, pInFrame2, test.ShouldResemble, pInFrame2)
 }
 
+func TestFakeSLAMGetPosition(t *testing.T) {
+	expectedComponentReference := ""
+	slamSvc := &SLAM{Name: "test", logger: golog.NewTestLogger(t)}
+
+	p, componentReference, err := slamSvc.GetPosition(context.Background(), slamSvc.Name)
+	test.That(t, err, test.ShouldBeNil)
+	test.That(t, componentReference, test.ShouldEqual, expectedComponentReference)
+
+	// test.ShouldBeBetween is used here as tiny differences were observed
+	// in floating point values between M1 mac & arm64 linux which
+	// were causing tests to pass on M1 mac but fail on ci.
+
+	test.That(t, p.Point().X, test.ShouldBeBetween, -0.005666600784453, -0.005666600784454)
```

Comment:
[opt] Use expected + tolerance values to make it easier to change in the future? (See integration tests)

Reply:
I'm changing this approach to use https://github.com/viamrobotics/rdk/pull/1965/files#diff-1380fdaf139fefcfce16ee38883d8c10aaf8416e3ea00c8999225c24d9861639R118 which was confirmed as idiomatic by the motion team here: https://viaminc.slack.com/archives/C01Q5S8V6TB/p1677787456102799?thread_ts=1677785113.482419&cid=C01Q5S8V6TB